### ISSUE TITLE
Cursor image does not show up

### ISSUE THREAD

**Issue by honggexjq** [alexzielenski/Mousecape#174]
I have tried to edit my cursor profile but only the "Arrow" image can successfully show up. Other images are not visible. What did I do wrong?
![image](https://user-images.githubusercontent.com/68488540/154885836-44f8b59f-6239-4801-9d3f-8f8bcd31443b.png)

**Comment by VisualisationExpo** [comment]
I don't think you're doing anything wrong. I can see that your cursor image is 32x32 pixels which is proper.
The only thing I would consider is setting the hotspot to 16, 16 for the Crosshair image. See if that makes a difference.

Have you been through the whole list of cursors and found that each and everyone doesn't work? I read your comment. I'm just asking for confirmation.

Try opening your cursor image in an graphics editor such as any free paint app or any other that you may already have bought. I would imagine Preview can open the image and then see if it's anything. Convert to PNG if it's all ICO files you're pasting in.

**Comment by honggexjq** [comment]
Thanks. Some pics indeed don't show up in preview. It's strange that they are working fine in Windows. Any idea how that happen?
![Uploading image.png…]()

**Comment by VisualisationExpo** [comment]
`.cur` files ought to work, but there could be instances where that's not the case.

In any case, I believe that a quick solution is to use rename all `.cur` files to `.png`via Automator.

Or if that's not the solution then you can perhaps share just one of the `.cur`files you're having issue with?

`.ani` files are a bit tougher to work around and there's a few solutions available that requires different tactics.

**Comment by honggexjq** [comment]
[3dgbeam.cur.zip](https://github.com/alexzielenski/Mousecape/files/8124704/3dgbeam.cur.zip)
Here is one of the cur file. Same result in preview - showing blank

**Comment by VisualisationExpo** [comment]
I agree, it's odd to say the least - works as you say on Windows, but not on macOS. Despite it being a regular .cur file.
I think that it's because it has a flag set from Windows that doesn't translate well on macOS - a permission flag.
Let me try setting some flags for the cursor your sent - I'll get back to you in a timely matter.
